pybamm/parameters/parameter_values.py

```diff
@@ -151,6 +153,42 @@ def update_from_chemistry(self, chemistry):
         if "sei" in chemistry:
             component_groups += ["sei"]
 
+        if "anode" in chemistry.keys():
```

Comment:
@tinosulzer @tlestang here are the warnings/errors you suggested, in case you want to take a look. Otherwise, let me know and I will merge the branch.